exercises/diffie-hellman/diffie_hellman_test.py

```diff
@@ -1,56 +1,40 @@
 import unittest
 
-import diffie_hellman
-
+from diffie_hellman import private_key, public_key, secret
 
 # Tests adapted from `problem-specifications//canonical-data.json` @ v1.0.0
 
-class DiffieHellmanTest(unittest.TestCase):
 
-    def test_private_key_is_in_range(self):
+class DiffieHellmanTest(unittest.TestCase):
+    def test_private_key_is_in_range_1_p(self):
         primes = [5, 7, 11, 13, 17, 19, 23, 29, 31, 37, 41, 43, 47]
         for i in primes:
-            self.assertTrue(1 < diffie_hellman.private_key(i) < i)
+            self.assertTrue(1 < private_key(i) < i)
 
-    # Can fail due to randomness, but most likely will not,
-    # due to pseudo-randomness and the large number chosen
     def test_private_key_is_random(self):
         p = 2147483647
         private_keys = []
         for i in range(5):
-            private_keys.append(diffie_hellman.private_key(p))
+            private_keys.append(private_key(p))
         self.assertEqual(len(set(private_keys)), len(private_keys))
 
     def test_can_calculate_public_key_using_private_key(self):
-        p = 23
-        g = 5
-        private = 6
-        expected = 8
-
-        actual = diffie_hellman.public_key(p, g, private)
-        self.assertEqual(actual, expected)
```

Comment:
I would prefer to keep the verbose variable declarations in this test for clarity on the usage of `public_key`